### PR TITLE
int validator and scope fix

### PR DESCRIPTION
**Pull request by Oneill38**
Bug fix for two issues:

1. Int validator would err if a single, regular int was passed along. Added `is_a?(Integer)` check to account for that

2. The `*Array(params(value))` was breaking scopes where only one argument was expected. Because there wasn't a version bump w/this I don't believe it ever got into Procore production. Rm'd it to return it back to what it originally was

### CODE REVIEW

**Comment by Oneill38** [comment]
ps i don't think that Gemfile.lock needs to be in there but idk how to pluck it out

File: lib/filterable/validators/valid_int_validator.rb [hunk 1]
```diff
@@ -15,6 +15,6 @@ def is_integer_array?(value)
   end
 
   def is_integer_or_range?(value)
-    !!(/\A\d+(...\d+)?\z/ =~ value)
+    value.is_a?(Integer) || !!(/\A\d+(...\d+)?\z/ =~ value)
```
Comment (HParker) [inline lib/filterable/validators/valid_int_validator.rb]: I am a little confused by this. don't integer params usually come through as string representations of the integer? Like `priority=1` becomes `"1"` in the controller.

Comment (dlcarter) [reply lib/filterable/validators/valid_int_validator.rb]: @HParker I think you're right.

Regardless, you _should_ be able to accomplish what you're trying to do simply by adding `.to_s` to the old line 18... so:
`!!(/\A\d+(...\d+)?\z/ =~ value.to_s)`

Comment (Oneill38) [reply lib/filterable/validators/valid_int_validator.rb]: heres the circle build: https://circleci.com/gh/procore/procore/181974#tests/containers/6
it chokes on trying to convert an integer to a string. Could be that the test is setup improperly, who knows. But to be safe, think I'll add the `to_s` in there to keep it cleaner

**Pull request merged**